recipe/recipe.yaml

```diff
@@ -20,6 +20,8 @@ source:
       - patches/0004-Add-conda-forge-include-dirs-to-list-of-include-dirs.patch
       # backport (relevant parts of) https://github.com/triton-lang/triton/pull/6928
       - patches/0005-fixes-definition-of-PY_SSIZE_T_CLEAN-macro.patch
+      # backport https://github.com/triton-lang/triton/pull/7384
+      - patches/0006-Handle-new-cuda-versions-7384.patch
```

Comment:
It looks like this was released with 3.5.0, which was merged into this feedstock last week. We can probably rebase on `main` and drop this patch.

The changelog https://github.com/triton-lang/triton/releases/tag/v3.5.0 mentions that PR is included.